perfkitbenchmarker/linux_benchmarks/fio_benchmark.py

```diff
@@ -460,6 +489,9 @@ def Run(benchmark_spec):
   stdout, stderr = vm.RobustRemoteCommand(fio_command, should_log=True)
   samples = fio.ParseResults(job_file_string, json.loads(stdout))
 
+  if collect_logs:
+    vm.PullFile(vm_util.GetTempDir(), '%s_*.log' % PKB_FIO_LOG_FILE_NAME)
```

Comment:
It's a little weird to be getting the tmp dir from two different sources. Suggest using `vm_util.VM_TMP_DIR` as above?